recipe/meta.yaml

```
@@ -7,14 +7,15 @@ package:
 
 source:
   url: https://github.com/matplotlib/basemap/archive/v{{ version }}.tar.gz
-  sha256: c35aa397be4de342a3078ec089094531609712abd78e3d11b92d792974fd8141
+  sha256: 42adbd039b4afbe93ca2fa32cfab456d4fd590cb8ed1c28e4aa94dcb957b58ac
   patches:
     # Update messages regarding installation of `basemap-data-hires`.
     - install_hires_instructions.patch
     - install_numpy_override.patch         # <--- This is a horible hack. Remove ASAP.
 
 build:
   number: {{ build_number }}
+  skip: True  # [py<37 or py>310]
```

It won't build for 3.11 with this selector.

---

yes, that is by intention.  it won't build right now as dependencies on python 3.11 aren't on main

---

Since cbc doesn't have 3.11 set, prefect won't attempt to build it anyway.
If you change to:
```suggestion
  skip: True  # [py<37]
```
At least we won't have to modify the feedstock when trying the 3.11 build

---

we can re-try.  but I ran into issue that it picked up python 3.11 on some testing